### PR TITLE
FEATURE: Add acceptedCurrencies to listing index.

### PR DESCRIPTION
**Pull request by ghost**
Partially completes #962

It was already in listing show pages but not the index.

### CODE REVIEW

**Comment by cpacia** [comment]
So I assume eventually we're going to deprecate the `coinType` field correct?

File: qa/listings.py [hunk 1]
```diff
@@ -44,20 +45,65 @@ def run_test(self):
...
+        if r.status_code == 404:
+            raise TestFailure("ListingsTest - FAIL: Listings get endpoint not found")
+        elif r.status_code != 200:
+            resp = json.loads(r.text)
+            raise TestFailure("ListingsTest - FAIL: Listings GET failed. Reason: %s", resp["reason"])
+        resp = json.loads(r.text)
+        if len(resp) != 1:
+            raise TestFailure("ListingsTest - FAIL: One listing should be returned")
+        if resp[0]["acceptedCurrencies"] != ["tbtc"]:
+            raise TestFailure("ListingsTest - FAIL: Listing should have acceptedCurrences")
```
Comment (placer14) [inline qa/listings.py]: Is there a benefit to checking the listing from another node? And if not, then I'd prefer a unit test in go. (Especially considering these may migrate depending on how we improve our QA, integration test setup?)

**Review by placer14 — APPROVED** [review]
One minor question. LGTM otherwise.

**Comment by tyler-smith** [comment]
@cpacia It's the coin that cryptocurrency listings are selling. So if you're selling BCH it'll be BCH.

File: qa/listings.py (hunk 1 above)
Comment (tyler-smith) [reply qa/listings.py]: There is a different code branch taken when getting your own listings versus someone elses so I wanted to test both.

Comment (tyler-smith) [reply qa/listings.py]: I went ahead and made some unit tests anyways.

**Comment by tyler-smith** [comment]
It's double tested so IDK why it says coverage is lower. Travis is passing.

**Review by cpacia — APPROVED** [review]

**Pull request merged**